### ISSUE TITLE
[Feature] Detect multiple classes

### ISSUE THREAD

**Issue by CodeWithDennis** [CodeWithDennis/filament-theme-inspector#9]
It would be great to have the option to customize which classes are being detected. Right now, it only looks for `fi-`, but it would be cool if that was something you could configure.

```php
->detect([
   'fi-', 
   'la-'
])
```

**Comment by CodeWithDennis** [comment]
#10
